astropy/cosmology/utils.py

```diff
@@ -1,5 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
+from __future__ import annotations
```

Comment:
Why?

Reply:
Removed from the git history. Legacy of a squash where a util had type hints.